server/testutils/factories/placementRequestDetail.ts

```diff
@@ -0,0 +1,51 @@
+import { Factory } from 'fishery'
+import { faker } from '@faker-js/faker/locale/en_GB'
+import { PlacementRequestDetail } from '../../@types/shared'
+import { DateFormats } from '../../utils/dateUtils'
+import personFactory from './person'
+import risksFactory from './risks'
+import userFactory from './user'
+import cancellationFactory from './cancellation'
+
+export default Factory.define<PlacementRequestDetail>(() => ({
+  id: faker.string.uuid(),
```

Comment:
Am I right in thinking a `PlacementRequestDetail` is just a `PlacementRequest` with a `cancellations` key/member? 

If so could this function be: 
```
export default Factory.define<PlacementRequestDetail>(() => ({
...placementRequestFactory.build(),
  cancellations: cancellationFactory.buildList(2)
  }))
  ``` 
  ? 
  If not ignore me!

Reply:
ooh yeah, great shout!